cmd/explaintest/r/new_character_set_invalid.result

```diff
@@ -5,7 +5,7 @@ insert into t values ('中文', 'asdf', '字符集');
 insert into t values ('À', 'ø', '😂');
 Error 1366: Incorrect string value '\xC3\x80' for column 'a'
 insert into t values ('中文À中文', 'asdføfdsa', '字符集😂字符集');
-Error 1366: Incorrect string value '\xC3\x80\xE4\xB8\xAD\xE6...' for column 'a'
+Error 1366: Incorrect string value '\xC3\x80' for column 'a'
```

Comment:
It's not same as MySQL.

Reply:
I think it is OK to have this difference?

Reply:
Em... Not sure. Seems we had a PR to fix it before. https://github.com/pingcap/tidb/pull/25087

Reply:
This PR does not introduce the garbled code as described in #25087. It extracts the error message from `ErrInvalidCharacterSet`(which should be in a well format).

The only difference is the number of invalid bytes displayed.